examples/ui.content/src/main/content/jcr_root/content/core-components-examples/library/adaptive-form/ruleeditor/.content.xml

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:fd="http://www.adobe.com/aemfd/fd/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0"
```

Comment:
Examples is deployed today in aemcomponents.dev which should only consists of component collaterals, ideally this collateral should be under content folder inside `it` inside this `https://github.com/adobe/aem-core-forms-components/tree/master/it/content/src/main/content/jcr_root/content/forms/sites`

Reply:
Done.